pkgs/data/fonts/sarasa-gothic/default.nix

```diff
@@ -0,0 +1,32 @@
+{stdenv, fetchurl, p7zip}:
+
+stdenv.mkDerivation (rec {
+  version = "0.5.2";
+  name = "sarasa-gothic-${version}";
+
+  package = fetchurl {
+    url = "https://github.com/be5invis/Sarasa-Gothic/releases/download/v${version}/sarasa-gothic-ttf-${version}.7z";
+    sha256 = "18ycw57k7yhrbb8njzhzk6x32xnjal61wr48qxkqy4lh9zfy0p22";
+  };
+
+  nativeBuildInputs = [ p7zip ];
+
+  unpackPhase = ''
+    7z x $package
+  '';
```

Comment:
Instead of fetching the already prebuilt release, can you try to make the build with Nix instead?

Reply:
What is the advantage of build fonts from source?

Reply:
Then it doesn't depend on sombody making the releases available. The source can be freely changed to a different one. E.g. In case the orginal one gets deleted.

Reply:
Sources can be vanished as well, but the tarball url here looks indeed strange.

Reply:
I'm thinking of git clones, if anybody forks/clones the repo the sources aren't lost.